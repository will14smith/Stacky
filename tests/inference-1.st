main () -> () { 1 test 2 test print print print print }

test i64 -> i64 str { { dup 1 + string } invoke }