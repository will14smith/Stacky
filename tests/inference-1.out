=== Output ===
3
2
2
1

=== Exit ===
0
